go/dpagg/sum_test.go

```diff
@@ -903,6 +903,207 @@ func TestThresholdedResultFloat64(t *testing.T) {
 	}
 }
 
+// Tests ComputeConfidenceInterval that If both bounds are non-negative, then trim any negative interval. Similarly, if both
```

Comment:
nit: `Tests that ComputeConfidenceInterval for int64 trims negative intervals if both bounds are non-negative; and trims positive intervals if both bounds are non-positive.`